deutsche-bank-adapter/src/main/java/de/adorsys/xs2a/gateway/service/impl/DeutscheBankPaymentService.java

```diff
@@ -105,12 +113,21 @@ public SinglePaymentInitiationInformationWithStatusResponse getSinglePaymentInfo
                         default:
                             throw new UnexpectedResponseStatusCodeException();
                     }
-                }
-        );
+                };
     }
 
     @Override
     public PaymentInitiationScaStatusResponse getPaymentInitiationScaStatus(String paymentService, String paymentProduct, String paymentId, String authorisationId, Headers headers) {
         throw new UnsupportedOperationException();
     }
+
+    @Override
+    public PaymentInitiationStatus getSinglePaymentInitiationStatus(String paymentProduct, String paymentId, Headers headers) {
+        requireSepaCreditTransfer(paymentProduct);
+        String uri = PAYMENTS_SEPA_CREDIT_TRANSFERS_URI + "/" + paymentId + "/status";
```

Comment:
could be moved to the constant, we have usage at line 82

Reply:
Changed.